framework/wazuh/core/cluster/dapi/dapi.py

```diff
@@ -32,10 +32,22 @@
 from wazuh.core.exception import WazuhException, WazuhClusterError, WazuhError
 from wazuh.core.wazuh_socket import wazuh_sendsync
 
-pools = common.mp_pools.get()
 
 authentication_funcs = {'check_token', 'check_user_master', 'get_permissions', 'get_security_conf'}
-events_funcs = {"send_event_to_analysisd"}
+events_funcs = {'send_event_to_analysisd'}
+
+node_info = wazuh.core.cluster.cluster.get_node()
+if node_info['type'] == 'master':
+    try:
+        pools = common.mp_pools.get()
+        pools.update({'authentication_pool': ProcessPoolExecutor(
```

Comment:
I had to move the assignment here because I couldn't obtain the cluster configuration in `framework/wazuh/core/common.py` (circular import)